src/types.ts

```
@@ -8,6 +8,14 @@ export interface VoiceOnboardingResponse {
 }
 
 export interface FaceOnboardingResponse {
+  data: {
+    onboard_result: FaceOnboardingResult;
+    document_auth?: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)